Gemfile

```diff
@@ -6,12 +6,11 @@ ruby '3.0.1'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem 'rails', '~> 7.0.1'
 
-# Use postgresql as the database for Active Record
-gem 'pg', '~> 1.1'
-
```

Comment:
Change this back the way it was, as we will handle the production deployment later

Reply:
Thank you for the feedback! I moved pg gem outside of production! ✌️